src/steth/BaseV2Vault.sol

```diff
@@ -133,19 +156,30 @@ abstract contract BaseV2Vault is sc4626, IFlashLoanRecipient {
 
     /**
      * @notice Sell any token for the "asset" token on 0x exchange.
-     * @param _token The token to sell.
+     * @param _tokenIn The token to sell.
+     * @param _tokenOut The token to buy.
      * @param _amount The amount of tokens to sell.
      * @param _swapData The swap data for 0xrouter.
      * @param _assetAmountOutMin The minimum amount of "asset" token to receive for the swap.
      */
-    function zeroExSwap(ERC20 _token, uint256 _amount, bytes calldata _swapData, uint256 _assetAmountOutMin) external {
+    function zeroExSwap(
+        ERC20 _tokenIn,
+        ERC20 _tokenOut,
+        uint256 _amount,
+        bytes calldata _swapData,
+        uint256 _assetAmountOutMin
+    ) external {
         _onlyKeeperOrFlashLoan();
 
+        if (!zeroExSwapWhitelist[_tokenOut]) revert InvalidOutToken();
```

Comment:
maybe say something like "not allowed" and pass the token address?

Reply:
[Chaged](https://github.com/lindy-labs/sandclock-contracts/pull/120/commits/311c30f7c5dde9b770b3c00ae59c0ed55c982870)